css/index.css

```diff
@@ -213,6 +213,120 @@
 }
 .box:hover {
   box-shadow: 1px 1px 10px 5px rgba(94, 94, 95, 0.2);
+} */
+.featured-products {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 1rem;
+}
+.featured-products p {
+  font-size: 2rem;
+  font-weight: 600;
+}
+.featured-brands {
+  display: flex;
+  gap: 2rem;
+  align-items: center;
+  justify-content: space-around;
+  margin-bottom: 5rem;
+}
+.brand {
+  position: relative;
+  width: 20rem;
+  height: 20rem;
+  padding: 1rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+.brand img {
+  width: 100%;
+}
+.brand p {
```

Comment:
Try using classes instead of the descendant selector.

Reply:
sure.. Thanks for the review and suggestion.

Reply:
Done